src/Compilers/Core/Portable/InternalUtilities/SpecializedCollections.Empty.Set.cs

```diff
@@ -10,8 +10,11 @@ internal partial class SpecializedCollections
         private partial class Empty
         {
             internal class Set<T> : Collection<T>, ISet<T>
+#if COMPILERCORE
```

Comment:
Which feedback?  In general we should avoid all `#ifdef` in the compiler (and Roslyn in general).  They contribute negatively to the maintenance and experience of the code base.